Dockerfile

```diff
@@ -31,16 +33,27 @@ for pkg in ${PKGS_TO_INSTALL}; do\n\
   }\n\
 done\n\
 yum -y install ${PKGS_TO_INSTALL}\n' >> /tmp/safe_yum.sh \
-&& /bin/bash /tmp/safe_yum.sh \
-&& yum clean all
+  && /bin/bash /tmp/safe_yum.sh \
+  && yum clean all \
+  && rm /tmp/safe_yum.sh
 
 RUN wget https://bootstrap.pypa.io/get-pip.py \
-  && python3.6 get-pip.py
+  && python3.6 get-pip.py \
+  && pip install -U pip pipenv uwsgi \
+  && rm -rf ~/.cache/pip
 
-#
-# This is needed by click: http://click.pocoo.org/5/python3/
-#
-ENV LC_ALL=en_US.UTF-8
-ENV LANG=en_US.UTF-8
+ENV LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    APPLICATION_ROOT="/"
+
+LABEL version="0.15.9" \
+  maintainer="arXiv <nextgen@arxiv.org>" \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="arXiv Base" \
+  org.label-schema.description="Base image for arXiv NG applications" \
+  org.label-schema.url="https://arxiv.github.io/arxiv-base" \
+  org.label-schema.vcs-url="https://github.com/arxiv/arxiv-base" \
+  org.label-schema.vendor="arXiv.org" \
+  org.label-schema.version="0.15.9"
```

Comment:
You may want to move the labels and envs that will remain unchanged up above yum install, so the layers should be preserved. Labels that change can be added towards the end. Also, if the version number is to be used in more than one places, we can declarer that as an independent ENV var and reuse that variable in different places.

Reply:
Cool; moved the labels in 87eff32 -- going to just go with one version tag for now